app/views/ht_institutions/index.html.erb

```diff
@@ -43,18 +43,44 @@
       <th data-sortable="true">Entity ID</th>
     </tr>
     </thead>
-    <% @institutions.other.each do |i| %>
+    <% @other_institutions.each do |i| %>
       <tr>
         <td><%= link_to i.inst_id, ht_institution_path(i.inst_id) %></td>
         <td><%= i.name %></td>
         <td><%= i.domain %></td>
         <td>
           <%= raw i[:us] ? '<i class="glyphicon glyphicon-ok"></i>' : '' %>
         </td>
-        <td><%= HTInstitutionPresenter.badge_for(i) %></td>
-        <td><%= i.entityID %></td>
+        <td><%= i.badge %></td>
+        <td><%= i.metadata_link %></td>
       </tr>
     <% end %>
   </table>
+
+  <br />
+
+  <h2> Add New Institution </h2>
+
+  <div data-turbolinks="false">
```

Comment:
should whack this div - was trying to debug a problem with the form below (it was missing `local:true`) that I was blaming on turbolinks